_docs/_latest/use/notifier.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Using Forseti Notifier
+order: 003
+---
+# { page.title }
+
```

Comment:
I think this needs a TODO here to denote that the document isn't complete.

Reply:
What else should go here?  I'll be happy to address it now.